src/Controller/MediaPolicyController.php

```diff
@@ -16,19 +16,18 @@ public function mediaPolicyAction(Request $request) : Response
 
         $arguments['title'] = 'Media policy';
 
-        $arguments['contentHeader'] = new ContentHeader($arguments['title'], null, 'eLife\'s media policy is designed to encourage high-quality, informed and widespread discussion of new research &mdash; before and after publication.');
+        $arguments['contentHeader'] = new ContentHeader($arguments['title'], null, 'eLife\'s media policy is designed to encourage high-quality, informed and widespread discussion of research that we peer-review and publish.');
 
         $arguments['body'] = [
             ArticleSection::basic($this->render(
-                new Paragraph('Prior to publication, authors are encouraged to present their findings to their peers, including at meetings and conferences; to deposit a copy of their manuscript with a preprint server (or other open repository or website); and to blog about their findings. None of these activities will affect consideration of a manuscript for publication in eLife.'),
-                new Paragraph('Authors are also welcome to speak to the media about their work at any time and may share advance copies of their manuscript with journalists as they prefer. They may also wish to ask their institutional press officers to help with advance promotion, once a manuscript is accepted. However, eLife encourages press officers to pitch studies widely at the time of publication only, rather than in advance, so that as many journalists as possible receive the story, and access to the full, peer-reviewed paper, at the same time.')
+                new Paragraph('eLife only peer-reviews and publishes articles that are made publicly available as preprints. We recently launched <a href="'.$this->get('router')->generate('inside-elife-article', ['id' => '741dbe4d', 'slug' => 'elife-s-new-model-open-for-submissions']).'">a new publishing model</a> that ends the accept/reject decision after peer review. Instead, papers invited for review are published as a Reviewed Preprint that contains public peer reviews and <a href="'.$this->get('router')->generate('inside-elife-article', ['id' => 'db24dd46', 'slug' => 'elife-s-new-model-what-is-an-elife-assessment']).'">an eLife assessment</a>. We also continue to publish research that was accepted after peer review as part of our traditional process.'),
```

Comment:
This is just a code review so I'm not going to be verifying that the copy is correct. My only feedback is that we shouldn't use the `slug` parameter when creating these internal links. The `slug` is not needed the user will be directed to the appropriate location.

This should be:

```
new Paragraph('eLife only peer-reviews and publishes articles that are made publicly available as preprints. We recently launched <a href="'.$this->get('router')->generate('inside-elife-article', ['id' => '741dbe4d']).'">a new publishing model</a> that ends the accept/reject decision after peer review. Instead, papers invited for review are published as a Reviewed Preprint that contains public peer reviews and <a href="'.$this->get('router')->generate('inside-elife-article', ['id' => 'db24dd46']).'">an eLife assessment</a>. We also continue to publish research that was accepted after peer review as part of our traditional process.'),
```

Reply:
Thanks! It's removed.